.github/workflows/deploy_kube.yaml

```
@@ -0,0 +1,178 @@
+name: Deploy production on Kube
+
+on:
+  workflow_dispatch: ~
+  schedule:
+    - cron: '30 7 * * 1,2,3,4,5' # At 08:30 (UTC, => 08:30 / 09:30 in Europe/Paris depending on the DST), each day of the week.
+  push:
+    branches:
+      - kube
+
+jobs:
+
+  build-production:
+    name: '🚧  Build 🚀'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    environment:
+      name: production
+      url: https://www.elao.io #${{ vars.WEBSITE_URL }}
```

❔ **question:** A remettre ? Tu as eu un soucis avec cette var ?

```suggestion
      url: ${{ vars.WEBSITE_URL }}
```

---

pas de soucis, je voulais pas la modifier juste pour mes tests